lecture/views.py

```
@@ -31,19 +67,46 @@ def lecture_home(request: HttpRequest) -> HttpResponse:
+    slug = request.GET.get('slug')  # Get the slug from the query parameter
+
+    course = get_object_or_404(Courses, slug=slug)
+    lectures = Lecture.objects.filter(course=course).prefetch_related('video_files')
+
+    return render(request, 'lecture/video_content.html', {
+        'course': course,
+        'lectures': lectures,
+        'type': "video"
+    })
+
+def stream_video(request, video_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `stream_video`

(ANN201)